Viewer/src/net/relinc/viewer/GUI/ExportGUI.java

```diff
@@ -474,9 +490,15 @@ private File writeConsoleExcelFileMakerJobFile(String path, int pointsToKeep) {
 		parametersString += "Dataset2$" + stressName + "$(" + stressUnit + ")" + "$" + trueEng + "\n";
 		parametersString += "Dataset3$" + strainName + "$(" + strainUnit + ")"  + "$" + trueEng + "\n";
 		parametersString += "Dataset4$" + strainName + " Rate$(" + strainRateUnit + ")" + "$" + trueEng + "\n";
-
+		if (faceForcePresent)
+		{
+            parametersString += "Dataset5$ " + "Front face force(" + faceForceUnit + ")$\n";
+            parametersString += "Dataset6$ " + "Back face force(" + faceForceUnit + ")$";
+        }
```

Comment:
SurePulse uses tabs, please convert to tabs. Should have a linter that runs automatically on the test server